.github/workflows/regenerate_rubocop_todos.yml

```diff
@@ -0,0 +1,33 @@
+name: Regenerate Rubocop TODOs
+on:
+  push:
+    branches:
+      - depfu/update/srcapi/rubocop-*
+permissions:
+  contents: write
```

Comment:
Hm I'm not sure about the security implications of this. In which context is this running? What if someone makes a PR from a fork from the branch `depfu/update/srcapi/rubocop-i-hack-you`. Won't this action hand out the keys to the kingdom?

Reply:
Ups... yes. Let's study them.